### ISSUE TITLE
Translation not applied on Kubuntu 20.04

### ISSUE THREAD

**Issue by Qik000** [lwindolf/liferea#846]
Translation is not applied fully (some things are translated while others are not) on Kubuntu 20.04, on my Ubuntu 18.04.4 machine translations work fine (Polish translation)

**Comment by lwindolf** [comment]
Do you use the Ubuntu package? 

Do you use the same Liferea version on 18.04 as on 20.04?

**Comment by Qik000** [comment]
> Do you use the Ubuntu package?
> 
> Do you use the same Liferea version on 18.04 as on 20.04?

Yes and yes

**Comment by lwindolf** [comment]
That's strange. I fear I cannot help here. I'd suggest to open a Kubuntu issue.

**Comment by Skif-off** [comment]
@lwindolf 
If it works for you,  can you update all po-files on your side?

I see that some strings from `glade/*.ui` marked as old strings and not used (with `#~`): , "_Subscriptions", "_Feed", "_Item", "_View", "Update _All" and so one. I tried `make update-po`: I saw `glade/*.ui` in POTFILES, but with the same result.

P.S. I use Xubuntu 20.04.

**Comment by lwindolf** [comment]
@Skif-off You mean for git master? Sure.

**Comment by Skif-off** [comment]
@lwindolf 
> You mean for git master?

Yes.

I tried to build some verions as is and with `make update-po` before `make`. Simple check: does the main menu and toolbar have a translation? I saw a strange result:
```
--------------------------------------------------------------
Ver.   | as is 1    | as is 2    | update-po 1 | update-po 2 |
-------|------------|------------|-------------|-------------|
1.12.0 | translated | translated | translated  | translated  |
1.12.1 | translated | translated | translated  | translated  |
1.12.2 | translated | translated | translated  | translated  |
1.12.3 | translated | translated | no          | no          |
1.12.4 | translated | translated | no          | no          |
1.12.5 | translated | translated | no          | no          |
1.12.6 | no         | translated | no          | no          |
1.12.7 | no         | no         | no          | no          |
1.12.8 | no         | no         | no          | no          |
1.13.0 | translated | translated | no          | no          |
1.13.1 | translated | translated | no          | no          |
master |            | translated |             | no          |
--------------------------------------------------------------
```
where:
1. as is 1: from `liferea-VERSION.tar.bz2` (i.e. with `./configure`).
2. as is 2: from `vVERSION.tar.gz` (i.e. with `./autogen.sh`).
3. update-po 1: from `liferea-VERSION.tar.bz2` (i.e. with `./configure`).
4. update-po 2: from `vVERSION.tar.gz` ( i.e. with `./autogen.sh`).

Additional info: Automake 1.15.1, GNU Make 4.2.1, GNU gettext 0.19.8.1, intltool 0.51.0 and GCC 9.3.0.

**Comment by Skif-off** [comment]
@Qik000
Try to install Liferea and update from Groovy (download and install deb-files manually):
https://packages.ubuntu.com/groovy/liferea
https://packages.ubuntu.com/groovy/liferea-data
Current version of this packages are 1.13.1-1 and this version works on Xubuntu 20.04.

**Comment by Skif-off** [comment]
@lwindolf
I found two typos in `POTFILES.in`:
```
[tyep: gettext/glade]glade/liferea_menu.ui
[tyep: gettext/glade]glade/liferea_toolbar.ui
```
"tyep" instead of "type". Fix it, please.

**Comment by lwindolf** [comment]
Thanks for spotting this!!!

**Comment by lwindolf** [comment]
Fix was released with 1.12.9
